### ISSUE TITLE
Expo/React-Native support

### ISSUE THREAD

**Issue by velsa** [hebcal/hebcal-es6#18]
Thank you very much for making hebcal ES6 compatible!
I am trying to use it in my Expo/React Native project, but, unfortunately, the package is crashing with various errors. E.g. 

`TypeError: undefined is not an object (evaluating '_core.HebrewCalendar.calendar')`

You can see it yourself by installing expo, doing expo init, and then pasting some of your demo code into the `App.tsx`.

Any help is greatly appreciated!

**Comment by mjradwin** [comment]
I'm not an Expo or webpack expert at all, but I can reproduce something similar to your problem. Can you help to suggest changes to the `package.json` so webpack does the right thing?

The error I get looks like this:

```
TypeError: Cannot read property 'lookup' of undefined
```

From this context:

```
  896 |   year: 1981,
  897 |   isHebrewYear: false,
  898 |   candlelighting: true,
> 899 |   location: _hebcal_core__WEBPACK_IMPORTED_MODULE_5__["Location"].lookup('San Francisco'),
      |                                                                   ^ 
  900 |   sedrot: true,
  901 |   omer: true
  902 | };
```

I can make the error go away by changing this line:

```js
import {HebrewCalendar, HDate, Location, Event} from '@hebcal/core';
```

To this:

```js
import {HebrewCalendar, HDate, Location, Event} from '@hebcal/core/dist/index.mjs';
```

Which is forcing the import statement to import from the ES6 module (`dist/index.mjs`, as declared in `@hebcal/core`'s `package.json` file). I am guessing that the former is picking up default `main` file, `dist/index.js`, which is a commonjs module, or the `browser` file, `dist/bundle.js`, which is an `iife` file.

What hint could I give webpack to cause it to prefer the proper ES6 module?

**Comment by mjradwin** [comment]
Update: this problem should be resolved in `@hebcal/core@3.3.0`

I did a bit of research on webpack and found that it prefers `package.json` entries in this order: `browser`, `module`, `main`. It's trying to use our `iife` file `bundle.js` as a module but it's not designed to be a proper module!

Instead, we have removed the `browser` key from the `package.json` so webpack will prefer the `index.mjs` file as it should.
